src/labels/CustomLabels.labels

```diff
@@ -1326,6 +1326,30 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>The total Amount of all Closed Won Opps is not equal to the RD Amount.</shortDescription>
         <value>The combined Amount of all related Closed Won and Pledged Opportunities must match the Amount of a Fixed Length Recurring Donation.</value>
     </labels>
+    <labels>
+        <fullName>RD_ValidateAccountAndContactMissing</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Recurring Donations data if contact and account are missing</shortDescription>
+        <value>You must select an Organization or Contact</value>
+    </labels>
+    <labels>
+        <fullName>RD_ValidateContactMissingForAccount</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Recurring Donations data if contact is missing</shortDescription>
+        <value>You must select a contact for this Household Account</value>
+    </labels>
+    <labels>
+        <fullName>RD_ValidateIfContactNotRelatedWithAccount</fullName>
```

Comment:
These two errors (`RD_ValidateContactMissingForAccount` and `RD_ValidateIfContactNotRelatedWithAccount`) feel like they could just be a single message (`RD_ValidateIfContactNotRelatedWithAccount`). What do you think @sam-knox?

Reply:
> These two errors (`RD_ValidateContactMissingForAccount` and `RD_ValidateIfContactNotRelatedWithAccount`) feel like they could just be a single message (`RD_ValidateIfContactNotRelatedWithAccount`). What do you think @sam-knox?

@force2b and @sam-knox , I have only two labels now - renamed as RD_DonorIsRequired and RD_ContactMustBelongToAccount. Please let me know if the values looks fine or needs to be modified. Since we both one label for two scenarios, one for both account/contact missing and the other is for the contact missing, we need some general error message here.

Reply:
@meena-sfdc - I added my suggested error message text to the ACs in the Work Item.